packages/block-library/src/query/edit/query-content.js

```diff
@@ -70,6 +71,7 @@ export default function QueryContent( {
 			?.posts_per_page;
 
 		return {
+			isNavigationMode: _isNavigationMode(),
```

Comment:
Not sure if this is best way to test for write mode inside a block?

Or should we be rather testing for `isSectionBlock`?